libsolidity/analysis/TypeChecker.cpp

```diff
@@ -369,16 +369,16 @@ void TypeChecker::endVisit(InheritanceSpecifier const& _inheritance)
 	solAssert(base, "Base contract not available.");
 
 	if (m_scope->contractKind() == ContractDefinition::ContractKind::Interface)
-		m_errorReporter.fatalTypeError(_inheritance.location(), "Interfaces cannot inherit.");
+		m_errorReporter.typeError(_inheritance.location(), "Interfaces cannot inherit.");
 
 	if (base->isLibrary())
-		m_errorReporter.fatalTypeError(_inheritance.location(), "Libraries cannot be inherited from.");
```

Comment:
Please either leave this fatal or try to crash the compiler in such a situation :-)